<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFunction</class>
 <widget class="QDialog" name="AddFunction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add member function...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/resources/RES/ffficons/sum.png</normaloff>:/resources/RES/ffficons/sum.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="fnL">
       <property name="text">
        <string>Function name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="returnL">
       <property name="text">
        <string>Return type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QComboBox" name="varType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>5</number>
         </property>
         <item>
          <property name="text">
           <string>Integer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Real</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>String</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Boolean</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Class</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>No return type</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectClass">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Select class...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="argumentsAddRemove" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Arguments:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="removeButton">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/resources/RES/ffficons/delete.png</normaloff>:/resources/RES/ffficons/delete.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="AddButton">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/resources/RES/ffficons/add.png</normaloff>:/resources/RES/ffficons/add.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="ArgumentList">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <column>
      <property name="text">
       <string>Argument name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Argument type</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="advancedButton">
       <property name="text">
        <string>Advanced...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okbutton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelbutton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelbutton</sender>
   <signal>clicked()</signal>
   <receiver>AddFunction</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
